src/core.jl

```diff
@@ -45,12 +45,23 @@ macro required(keywords...)
 end
 
 """
-    tf_version()
+    tf_version(kind=:julia)
 
-Return the version number of the C tensorflow library.
+Return the version number of the tensorflow library.
+
+If `kind` is `:julia`, return the version of the TensorFlow binary loaded
+into the Julia process.
+
+If `kind` is `:python`, return the version of the Python API.
 """
-function tf_version()
-    res = @tfcall(:TF_Version, Cstring, ()) |> unsafe_string
+function tf_version(; kind=:julia)
```

Comment:
Possibly this should be 3 options:

 - `:C` for getting the shared library version (or `:backend`), i.e. current behavour of `:julia`
 - `:python` for getting python version
 - `:julia` for gettings version of TensorFlow.jl (just call `Pkg.installed("TensorFlow")`)